src/seedu/addressbook/AddressBook.java

```diff
@@ -203,13 +203,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        executeUserCommand();
+    }
+
+	public static void executeUserCommand() {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.